common/Log.hpp

```diff
@@ -260,13 +272,13 @@ static constexpr std::size_t skipPathPrefix(const char (&s)[N], std::size_t n =
 
 #ifdef __ANDROID__
 
-#define LOG_LOG(LOG, NAME, PRIO, LVL, STR)                               \
+#define LOG_LOG(LOG, NAME, LVL, STR)                                     \
     ((void)__android_log_print(ANDROID_LOG_DEBUG, "coolwsd", "%s %s", LVL, STR.c_str()))
 
 #else
 
-#define LOG_LOG(LOG, NAME, PRIO, LVL, STR)                           \
-    LOG.log(Poco::Message(NAME, STR, Poco::Message::PRIO_##PRIO))
+#define LOG_LOG(LOG, NAME, LVL, STR)                                     \
+    LOG.log(Poco::Message(NAME, STR, static_cast<Poco::Message::Priority>(LVL)))
```

Comment:
We really haven't removed any dependencies here, we just made our dependency implicit and fragile.

Sorry, I don't see the benefit of redefining the Priority enum and then casting it. We are far better off using the enum from Poco directly and explicitly. This kind of a change is how we can shoot ourselves in the foot, for no gains whatsoever (even this code is more complex than the old).

Reply:
@Ashod I want to remove the Poco headers from our code - so we don't accidentally use new features there, and we have better control over what we use. I'd like to kill Poco from the implementation too. We can have a no-op mapping function inside the Log.cxx if you're concerned that the numbers may drift =) once we start logging with Poco performance is terrible anyway - several redundant system calls per log message I'm afraid.